### DISCUSSION TITLE
New Nacht der untoten High rounds & Records

### DISCUSSION THREAD

**Discussion by notrickzdumbo** [nzp-team/nzportable#282] [Records and Speedruns]
As of the latest nightly a brand new and improved Nacht der untoten map was released! it is more faithful to the Call of Duty World at War map that started it all and has all the weapons from it such as the flamethrower included in the box.

I will start the high rounding discussion with my own record on it which is round 34. I used the Flamethrower, Thompson and Ray Gun alongside training at the end of the help room (without opening the help door). Good luck to anyone who tries to beat it!

![fte-20230402194424-0](https://user-images.githubusercontent.com/98246579/229387276-eb443715-eae2-43d4-ad60-e22593ca1b81.png)

**Comment by KingDoge176** [comment]
imma try to beat it

**Reply by thinzpfan** [reply]
same

**Comment by thinzpfan** [comment]
ty for the strat I need a 20 on this as I'm going for a 20 on each nzp team map
